src/components/Footer/Footer.js

```diff
@@ -1,7 +1,42 @@
 import React from 'react';
+import './Footer.scss';
+import { FOOTER_LIST, images } from './footerData';
 
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <footer className="Footer">
```

Comment:
형경님! 
최상위 클래스 명 = 컴포넌트 명 이지만,
컨벤션 자체는 카멜케이스로 통일 시켜주세요!
```suggestion
    <footer className="footer">
```